### ISSUE TITLE
<user_path>third_party/eigen3/unsupported/Eigen/CXX11/Tensor:4:10: 'unsupported/Eigen/CXX11/Tensor' file not found

### ISSUE THREAD

**Issue by Heng14** [yjmade/ios_camera_object_detection#3]
@yjmade hi, i have an error that the line in Tensor file #include "unsupported/Eigen/CXX11/Tensor" cause an error: 'unsupported/Eigen/CXX11/Tensor' file not found. but it didn't have such error when i run the simple in official ios_examples. could you please help me?

**Comment by StarRain-L** [comment]
这个问题是路径不对

**Comment by brianantonelli** [comment]
In header search paths replace `"$(SRCROOT)/../../makefile/downloads/eigen-latest"` with `"$(SRCROOT)/../../makefile/downloads/eigen"`

**Comment by DAYHU** [comment]
@brianantonelli Thank you very much but I met another problem after changing the path as below:

Undefined symbols for architecture x86_64:
  "_cblas_sgemm", referenced from:
      tensorflow::Conv2DUsingGemmOp<float, tensorflow::(anonymous namespace)::Im2ColConvFunctor<float, float, float, FastGemmFunctor<float, float, float> > >::Compute(tensorflow::OpKernelContext*) in libtensorflow-core.a(conv_ops_using_gemm.o)
      tensorflow::FusedResizeConv2DUsingGemmOp<float, tensorflow::(anonymous namespace)::FusedResizeAndPadConvFunctor<float, float, float, FastGemmFunctor<float, float, float>, (tensorflow::(anonymous namespace)::SamplingMode)0>, true>::Compute(tensorflow::OpKernelContext*) in libtensorflow-core.a(conv_ops_fused.o)
      tensorflow::FusedResizeConv2DUsingGemmOp<float, tensorflow::(anonymous namespace)::FusedResizeAndPadConvFunctor<float, float, float, FastGemmFunctor<float, float, float>, (tensorflow::(anonymous namespace)::SamplingMode)1>, false>::Compute(tensorflow::OpKernelContext*) in libtensorflow-core.a(conv_ops_fused.o)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

**Comment by Heng14** [comment]
@brianantonelli @StarRain-L thank you very much and that works!

**Comment by Heng14** [comment]
@yjmade @brianantonelli  I have applied it on iPhone 7 successfully but it cannot detect any objects. It have not any response but just open the camera. I have no idea where the problem is. Could you please help me ? thank you

**Comment by yjmade** [comment]
any log from the console?

**Comment by Heng14** [comment]
@yjmade no, just like a camera and no detection or response.

**Comment by brianantonelli** [comment]
I've had similar results on iPhone 7 on iOS 10. I cannot run it with the debugger attached as it constantly crashes.. Ive had a few times when Ive run the app and it crashed over and over and then suddenly it worked correctly and did image detection. I tried running `optimize_for_inference` on the graph to see if that would help but it did not, I think recompiling Tensorflow for production might resolve some of these issues. 

Here is proof tho that it does occasionally work: 
![image_uploaded_from_ios](https://user-images.githubusercontent.com/454010/27658360-c20eb0da-5c1d-11e7-9a9a-c672d64d7198.jpg)

**Comment by Heng14** [comment]
@brianantonelli  yeah, I have the same problem. It always crashes out or no response. I have no idea what to do.
